ground-core/docker/config.yml

```diff
@@ -0,0 +1,17 @@
+dbType: neo4j
```

Comment:
For docker to build the image it needs the config file to be in the same folder as the docker file. I can't think of a way to include the normal file.